spec/travis/addons/handlers/pusher_spec.rb

```diff
@@ -43,7 +43,7 @@
           expect(payload['queue']).to   eq('pusher-live')
           expect(payload['class']).to   eq('Travis::Async::Sidekiq::Worker')
           expect(payload['args'][3]).to be_a(Hash)
-          expect(payload['args'][4]).to eq(event: event, user_ids: [5])
+          expect(payload['args'][4]).to eq(event: event, user_ids: [user.id])
```

Comment:
This was failing as the id was `8`? Seems like using the id directly makes more sense.

Reply:
totally. 👍